test/library/draft/arrow/ArrowParquet.chpl

```diff
@@ -0,0 +1,60 @@
+module ArrowParquet {
+  require "ArrayDecl.chpl", "ParquetDecl.chpl", "ArrowRecordDecl.chpl";
+  use ArrayDecl, ParquetDecl, ArrowRecordDecl;
+  use SysCTypes, CPtr;
+
+  proc writeTableToParquetFile(table: GArrowTable, path: string) {
+    var error: GErrorPtr;
+    var writer_properties: GParquetWriterProperties = gparquet_writer_properties_new();
+    var writer: GParquetArrowFileWriter = gparquet_arrow_file_writer_new_path(
+                                                  garrow_table_get_schema(table),
+                                                  path.c_str(), 
+                                                  writer_properties,
+                                                  c_ptrTo(error));
+  if(isNull(writer)){
+      g_print("failed to initialize writer: %s\n", error.deref().message);
+      g_error_free(error);
+      exit(EXIT_FAILURE);
+    }
+    var success: gboolean = gparquet_arrow_file_writer_write_table(writer,
+                                                      table ,
+                                                      10 : guint64,
+                                                      c_ptrTo(error));
+    if(!success){
+      g_print("failed to write table: %s\n", error.deref().message);
+      g_error_free(error);
+      exit(EXIT_FAILURE);
```

Comment:
Similar to the error reporting in the previous file, this code is copy pasted 3 times here, maybe would make sense in a helper function.